docs/docker.md

```diff
@@ -2,43 +2,132 @@
 
-With docker-compose, run `docker-compose run app /venv/bin/umap createsuperuser`
+SECRET_KEY = '!!secretsecret!!'
+INTERNAL_IPS = ('127.0.0.1', )
+ALLOWED_HOSTS = ['*', ]
 
+DEBUG = True
+COMPRESS_ENABLED = True
+COMPRESS_OFFLINE = True
+LANGUAGE_CODE = 'en'
 
-### Custom settings
+# Set to False if login into django account should not be possible. You can                                                                                                              
+# administer accounts in the admin interface.                                                                                                                                            
+ENABLE_ACCOUNT_LOGIN = True
+AUTHENTICATION_BACKENDS = (
+    'django.contrib.auth.backends.ModelBackend',
+)
 
-Some basic settings are available through env vars (see https://github.com/umap-project/umap/blob/master/umap/settings/base.py),
+# Enables a banner letting users know this site is not for production use                                                                                                                
+UMAP_DEMO_SITE = True
+
+# Whether to allow non authenticated people to create maps.                                                                                                                              
+UMAP_ALLOW_ANONYMOUS = True
+```
+
+Some basic settings are available through env vars (see https://github.com/umap-project/umap/blob/master/umap/settings/base.py) and can be defined right in the docker-compose file,
 but if you need more custom ones (like custom OAuth configuration), the easiest
-way is to push a [settings file](settings.md) to `/etc/umap/umap.conf`.
+way is to put them in a [settings file](settings.md) and mount it to `/etc/umap/umap.conf`.
+
+---
+
+#### Quick sidebar about docker and docker-compose
+A multi-platform tutorial on [installing docker](https://docs.docker.com/get-docker/) and docker-compose is outside the scope of this howto, but if you are on a recent Ubuntu, this is all you need to install both:
+
+```bash
+sudo groupadd docker 
+sudo snap install docker
+sudo usermod -aG docker $USER
+# EXIT and log back in to pickup the docker group membership, then test with
+docker run hello-world
+```
+
+Current versions of docker compose are integrated into the docker command and launch with `docker compose` (two words) -- older versions use a separate binary named `docker-compose`. Either one will work for this. 
+
+---
+
+### Initial launch
+
+Start the server with 
+```bash
+docker compose up
+```
+... and let it run some initial setup until the output quiesces with a message about spawning uWSGI workers. Because there is a race between the time the app tries to connect to the DB and when the DB is actually ready, you might see a few exceptions/erros about 'psychopg' being unable to connect. This should sort itself out as the app retries. 
```

Comment:
```suggestion
... and let it run some initial setup until the output quiesces with a message about spawning uWSGI workers. Because there is a race between the time the app tries to connect to the DB and when the DB is actually ready, you might see a few exceptions/errors about 'psycopg' being unable to connect. This should sort itself out as the app retries. 
```

👍 Hopefully it will be mitigated by https://github.com/umap-project/umap/pull/1314